client/reader/sidebar/index.jsx

```diff
@@ -228,10 +219,12 @@ export class ReaderSidebar extends React.Component {
 									{ 'sidebar-activity__likes': true }
 								) }
 							>
-								<a href="/activities/likes" onClick={ this.handleReaderSidebarLikeActivityClicked }>
-									<MaterialIcon icon="star_border" />
-									<span className="menu-link-text">{ this.props.translate( 'My Likes' ) }</span>
-								</a>
+								<SidebarItem
+									label={ this.props.translate( 'My likes' ) }
```

Comment:
:disappointed: 61 existing translations will be lost with this change.
If the string must change, the following option has already been translated **30** times:
`translate( 'Go to My Likes' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).